test/Provider.test.js

```diff
@@ -1,6 +1,8 @@
 import React from "react"
-import { Provider } from "../src"
+import { observable } from "mobx"
+import { observer, Provider } from "../src"
 import { render } from "@testing-library/react"
+import renderer, { act } from "react-test-renderer"
```

Comment:
I don't follow why to use this one when line above you have RTL. Let's not mix two approaches in a single test file please :)

Reply:
It is already done :) I will do this step by step.

Reply:
UPD: This is the final version of the PR.